cloudify_starlingx/workflows/utils.py

```diff
@@ -82,3 +80,23 @@ def update_runtime_properties(instance,
                                          instance.state,
                                          props,
                                          instance.state + 1)
+
+
+def desecretize_client_config(config):
```

Comment:
this part little bit confused me, why you look for `get_secret` ?

Reply:
When working in the rest client intrinsic functions are not resolved.